src/include/daos_types.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2015-2021 Intel Corporation.
+ * (C) Copyright 2015-2022 Intel Corporation.
```

Comment:
I think maybe you did a merge commit and the githook updated copyrights for files outside of your PR. This is an annoying bug but I haven't found a way to modify the githook script to detect merge commits. You just have to remember to use `NO_PRE_COMMIT=true git commit` when committing the merge.